### PR TITLE
WIP: new adjustments

### PR DESCRIPTION
**Pull request by vpodlnk**
Related to another request https://github.com/PlayQ/tg/pull/273

### CODE REVIEW

File: d4s/src/main/scala/d4s/DynamoInterpreter.scala [hunk 1]
```diff
@@ -160,6 +160,7 @@ object DynamoInterpreter {
   private[this] implicit final class ThrowableDynamoOps[F[+_, +_], A](private val f: F[Throwable, A]) extends AnyVal {
     def logThrowable(operation: String, tableName: String)(implicit F: BIOError[F], log: LogBIO[F]): F[Throwable, A] = {
       f.tapError {
+        case _: ConditionalCheckFailedException => F.unit
```
Comment (aliubymov-playq) [inline d4s/src/main/scala/d4s/DynamoInterpreter.scala:163]: I think we should add a debug log here instead of just a unit.

Comment (aliubymov-playq) [reply d4s/src/main/scala/d4s/DynamoInterpreter.scala]: @vpodlnk Can we also change TableAlreadyExists exception to debug? 
And maybe ResourceNotFound. To reduce logs during prefixed table creation. 
@neko-kai what do you think?

Comment (neko-kai) [reply d4s/src/main/scala/d4s/DynamoInterpreter.scala]: I think TableAlreadyExists should be logged as debug only if it's going to be ignored (e.g. during DDL up), but not if it gets thrown up the stack. e.g. add another combinator to ignore these and use it in TableManager

Comment (vpodlnk) [reply d4s/src/main/scala/d4s/DynamoInterpreter.scala]: So, as I understood, @aliubymov-playq you've opened a separated PR for this (https://github.com/PlayQ/d4s/pull/28), should I revert this particular line?

**Comment by vpodlnk** [comment]
@neko-kai , @aliubymov-playq , mates, if it looks good for you, I'd like to merge those changes.
What do you think?

**Pull request merged**